app/assets/stylesheets/application.scss

```diff
@@ -1,3 +1,6 @@
+@import "lib/govuk-frontend/govuk/all";
+@import "lib/hmrc-frontend/hmrc/all";
```

Comment:
There is a path missing which causes 404 to be returned by some assets calls. Adding this will fix (need to change the URL to match this service):
https://github.com/hmrc/emcs-tfe-create-movement-frontend/blob/1f8a52f90c1d33b9036516ef458abb56f0fffccf/app/assets/stylesheets/application.scss#L1-L2